src/pymor/algorithms/timestepping.py

```diff
@@ -93,17 +153,80 @@ class ImplicitEulerTimeStepper(TimeStepper):
     def __init__(self, nt, solver_options='operator'):
         self.__auto_init(locals())
 
-    def solve(self, initial_time, end_time, initial_data, operator, rhs=None, mass=None, mu=None, num_values=None):
-        return implicit_euler(operator, rhs, mass, initial_data, initial_time, end_time, self.nt, mu, num_values,
-                              solver_options=self.solver_options)
+    def estimate_time_step_number(self, initial_time, end_time):
+        return self.nt + 1
```

Comment:
I'd expect this to only return `nt`.

Reply:
Maybe it's better to call the function `estimate_num_values`?

Reply:
But it estimates the number of time steps plus 1, not the number of returned values, right?